torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -468,10 +468,35 @@ void XLAGraphExecutor::MaybeDumpGraph(std::string name,
   }
 }
 
-XLAGraphExecutor::ComputationCache* XLAGraphExecutor::GetComputationCache() {
+XLAGraphExecutor::ComputationCache* createComputationCache() {
```

Comment:
nit. put this in a unnamed namesmape at the top, if not exposed via the 
 header file.

Reply:
Also, should we call it `CreateComputationCache`?